packages/tools/pie-icons-webc/README.md

```diff
@@ -20,37 +22,43 @@ Add the module to your project
 yarn add @justeattakeaway/pie-icons-webc
 ```
 
+
 ### Applications
 
 ```js
 // Only import what you need!
-import { IconAppRestaurant, IconAppRestaurantLarge } from '@justeattakeaway/pie-icons-webc';
+import '@justeattakeaway/pie-icons-webc/icons/IconAppRestaurant';
+
+// Note, we don't recommend referencing named exports such as:
+// `import { IconAppRestaurant } from '@justeattakeaway/pie-icons-webc`
+// As this will require you to make a call to the icon so that it doesn't get tree-shaken by your build tool
 ```
+
 Within the context of a Lit app, that will look like:
 
-  ```js
-import { IconAppRestaurant } from '@justeattakeaway/pie-icons-webc';
-export { IconAppRestaurant }
-
-  export class IconButton extends LitElement {
-    render () {
-      return html`
-        <button>
-          <icon-app-restaurant size="xs" />
-        </button>`;
-    }
+```js
+import '@justeattakeaway/pie-icons-webc/icons/IconAppRestaurant';
+
+export class IconButton extends LitElement {
```

Comment:
I think this example might be improved by adding some text to show that it's not just a plain icon button, which we of course have a component for :smile:

Reply:
Fixed